.circleci/config.yml

```diff
@@ -22,6 +22,16 @@ workflows:
     jobs:
       - test
       - topic
+  weekly_link_check:
+    triggers:
+      - schedule:
+          cron: '30 12 * * 1' # Every Monday at 12:30pm UTC
```

Comment:
Nit: I'd suggest the comment have an EST time as well; it's hard for me to know what UTC time is locally (it's easy to convert from EST to PST).